app.py

```
@@ -730,11 +1043,19 @@ def pdf_to_text():
         if not user:
             return jsonify({'error': 'User not found'}), 404
         
-        # Check if the user has already used the free trial feature
-        if not user['subscription']['is_subscribed'] and user['features'].get('service_used', False):
-            print('Free Trial limit reached. Please subscribe to continue.')
-            return jsonify({'error': 'Trial limit reached. Please subscribe to continue.'}), 403
+        if not user.get('is_superadmin', False):    
+            # Check if the user has already used the free trial feature
+            if not user['subscription']['is_subscribed'] and user.get('first_job_uploaded', False):
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)